### ISSUE TITLE
Accessibility: Color contrast insufficient

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51895]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The sale text on the product images in the "You may be interested in..." section fails color contrast against the background color. See color contrast analysis: https://webaim.org/resources/contrastchecker/?fcolor=B3AF54&bcolor=FFFFFF

![Image](https://github.com/user-attachments/assets/f71caaaa-14c6-492a-99b5-ad6eefd33778)

### Component or Page: 

Shortcode: [woocommerce_cart] & Cart Classic Block

### Name affected element:

"Sale" text

### Relevant code snippets:

`<span class=""onsale"">Sale!</span>

.woocommerce span.onsale {
    background-color: var(--wp--preset--color--foreground, #b3af54);
    color: var(--wp--preset--color--background, #fff);
}`

### Recommended fix:

Change the colors to have a minimum color contrast of at least 4.5:1.
